infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultLogReplicationConfigAdapterForUpgradeStandby.java

```diff
@@ -0,0 +1,107 @@
+package org.corfudb.infrastructure.logreplication.infrastructure.plugins;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuStore;
+import org.corfudb.runtime.collections.TableOptions;
+import org.corfudb.runtime.collections.TxnContext;
+import org.corfudb.runtime.view.TableRegistry;
+import org.corfudb.utils.CommonTypes;
+import org.corfudb.utils.LogReplicationStreams;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * This class is created for LR rolling upgrade tests to serve as the plugin for Standby cluster to
+ * get the set of streams to replicate.
+ */
+public class DefaultLogReplicationConfigAdapterForUpgradeStandby implements ILogReplicationConfigAdapter {
+    private final Set<String> streamsToReplicate = new HashSet<>();
+    private static final String STREAMS_TEST_TABLE =
+        "StreamsToReplicateTestTable";
+    private static final String VERSION_TEST_TABLE = "VersionTestTable";
+
+    public static final int MAP_COUNT = 5;
+    public static final String TABLE_PREFIX = "Table00";
+    public static final String NAMESPACE = "LR-Test";
+    public static final String TAG_ONE = "tag_one";
+    private final int indexOne = 1;
+    private final int indexTwo = 2;
+    private final String ENDPOINT = "localhost:9001";
+    private String versionString = "version_latest";
+    private CorfuStore corfuStore;
+
+    public DefaultLogReplicationConfigAdapterForUpgradeStandby() {
+        CorfuRuntime runtime =
+            CorfuRuntime.fromParameters(CorfuRuntime.CorfuRuntimeParameters.builder()
+                .build())
+                .parseConfigurationString(ENDPOINT).connect();
+        corfuStore = new CorfuStore(runtime);
+    }
+
+    // Provides the fully qualified names of streams to replicate
+    @Override
+    public Set<String> fetchStreamsToReplicate() {
+        try {
+            corfuStore.openTable(NAMESPACE, STREAMS_TEST_TABLE,
+                LogReplicationStreams.TableInfo.class,
+                LogReplicationStreams.Namespace.class, CommonTypes.Uuid.class,
+                TableOptions.builder().build());
+        } catch (Exception e) {
+            // Just for wrap this up
+        }
+
+        if (corfuStore != null) {
+            try (TxnContext txn = corfuStore.txn(NAMESPACE)) {
+                Set<LogReplicationStreams.TableInfo> tables = txn.keySet(STREAMS_TEST_TABLE);
+                tables.forEach(table -> streamsToReplicate.add(table.getName()));
+                txn.commit();
+            }
+        } else {
+            for (int i = 1; i <= MAP_COUNT; i++) {
+                streamsToReplicate.add(NAMESPACE + "$" + TABLE_PREFIX + i);
```

Comment:
Can we declare `$` to a variable to overcome Codacy error - `The String "$" appears 3 times in the file.`?